test/provision_test.py

```diff
@@ -74,3 +74,18 @@ def test_delete_twice(cleanup):
 
     for b in BACKENDS.values():
         assert not b.exists()
+
+
+@pytest.mark.sudo
+@pytest.mark.xfail(reason="Test end with xfail", strict=True)
```

Comment:
With this this test will always xfail, while we want it to pass.

Reply:
Initially, my idea with this mark was to make sure it will xfail, hence the `strict=True`. That way, if it did not execute `pytest.xfail`, it would actually fail.

But let's try to make it pass instead.